lucida/template/java/TemplateClient/start-Template-client.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
```

Comment:
there isn't a better way to do this? i know this is how its done but not how it should necessarily be moving forward.